docker/deployer/Dockerfile

```diff
@@ -58,6 +58,7 @@ RUN --mount=type=cache,target=/root/.cache \
 
 # split install in two steps to leverage docker cache
 COPY . .
+RUN curl --proto '=https' --tlsv1.2 -sSf https://docs.swmansion.com/scarb/install.sh | sh -s -- -v 0.7.0
```

Comment:
are you sure it's fine to install scarb directly and not with asdf?

this installs version 0.7.0, required to compile pragma contracts, but we might need other versions later

i'm fine if this works now but we will need to rememebr this

Reply:
yeah I wasn't sure actually, asdf might be better in the future if we need a lot more different versions